weldx/__init__.py

```diff
@@ -13,6 +13,9 @@
         category=UserWarning,
     )
 
+# constants - should be imported first, no internal weldx deps
+from weldx.constants import WELDX_QUANTITY as Q_
```

Comment:
please note: the `weldx.constants` module should have no internal `weldx` dependencies and always be imported first to avoid circular importing troubles @vhirtham @marscher